logstash-filter-grok.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-filter-grok'
-  s.version         = '4.1.1'
+  s.version         = '4.1.2'
```

Comment:
I would consider the adding of the `timeout_scope` to be worthy of a minor bump, not a patch-level bump.